flink-runtime/src/main/java/org/apache/flink/runtime/state/VoidNamespace.java

```diff
@@ -56,13 +54,4 @@ public boolean equals(Object obj) {
     public String toString() {
         return getClass().getSimpleName();
     }
-
-    // ------------------------------------------------------------------------
-    //  Singleton serialization
-    // ------------------------------------------------------------------------
-
-    // make sure that we preserve the singleton properly on serialization
-    private Object readResolve() throws ObjectStreamException {
```

Comment:
This is wrong. https://docs.oracle.com/javase/7/docs/platform/serialization/spec/input.html#5903

Reply:
Thanks for pointing that out.
my bad, I reverted it